subgraph/bin/subgraphOperation.js

```diff
@@ -77,6 +77,11 @@ const networkMap = {
       graphNode: 'https://api.thegraph.com/deploy/',
       ipfs: 'https://api.thegraph.com/ipfs/',
     },
+    binance: {
```

Comment:
Maybe eventually we can move all this to the `@unlocl-protocol/networks` package!